content/influxdb/v2.0/reference/flux/language/string-interpolation.md

```diff
@@ -27,19 +27,14 @@ name = "John"
 // My name is John.
 ```
 
-{{% note %}}
-#### Flux only interpolates string values
-Flux currently interpolates only string values ([IMP#1775](https://github.com/influxdata/flux/issues/1775)).
-Use the [string() function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/type-conversions/string/)
-to convert non-string values to strings.
-
-```js
-count = 12
+String interpolation expressions must satisfy the Stringable constraint.
```

Comment:
- Add the "Stringable" constraint to this doc: /influxdb/v2.0/reference/flux/language/types/#type-constraints
- Link "Stringable constraint" to the new section